src_c/display.c

```diff
@@ -2273,7 +2278,8 @@ pg_toggle_fullscreen(PyObject *self, PyObject *_null)
                              1) != 0) {
                 return NULL;
             }
-            win = SDL_CreateWindow(state->title, wx, wy, w, h, 0);
+            flags &= ~SDL_WINDOW_FULLSCREEN_DESKTOP;
+            win = SDL_CreateWindow(state->title, wx, wy, w, h, flags);
             if (win == NULL) {
                 return RAISE(pgExc_SDLError, SDL_GetError());
             }
```

Comment:
The comment in this block of code says it's a hack for SDL 2.0.8 (which is a rather old version)... not sure whether it's relevant to this day? Would need someone running pygame on linux with x11 to check though